src/svg-renderer.js

```diff
@@ -232,6 +242,24 @@ class SvgRenderer {
         }
     }
 
+    /**
+     * Fix SVGs to match appearance in Scratch 2, which used nearest neighbor scaling for bitmaps
+     * within SVGs.
+     */
+    _transformImages () {
+        const imageElements = this._collectElements('image');
+
+        // For each image element, set image rendering to pixelated"
+        for (const elt of imageElements) {
```

Comment:
In the interest of maintenance I'd recommend pulling the `image-rendering` settings out into a common constant:
```suggestion
        const pixelatedImages = 'image-rendering: optimizespeed; image-rendering: pixelated;';
        for (const elt of imageElements) {
```